.github/workflows/run-match.yml

```diff
@@ -1,12 +1,26 @@
 name: Simulator match
 
 on:
-  push: {}
+  push:
+    branches:
+      - master
+  pull_request: {}
   release:
     types: [ created ]
 
 jobs:
+  skip_duplicates:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@v3.3.0
```

Comment:
Any reason to use this over https://github.com/styfle/cancel-workflow-action? It seems to be used more

Reply:
As far I can tell that one does not handle skipping commits with the same information, only cancelling previous runs.